gh-pages/src/content/description.tsx

```diff
@@ -0,0 +1,66 @@
+import { Anchor, ConnectedComponent, Panel, PanelBody, PanelFooter, PanelHeader } from 'msteams-ui-components-react';
+import * as React from 'react';
+
+export const Description: React.StatelessComponent = () => {
+  return <ConnectedComponent render={(props) => {
+      const { context } = props;
+      const { rem, font } = context;
+      const { sizes, weights } = font;
+
+      const styles = {
+        header: { ...sizes.title, ...weights.semibold },
+        section: { ...sizes.title2, marginTop: rem(1.4), marginBottom: rem(1.4) },
+        button: {marginRight: rem(0.5) },
+      };
+
+      return <Panel>
+        <PanelHeader>
+          <div style={styles.header}>Home</div>
+        </PanelHeader>
+        <PanelBody>
+          <div style={styles.section}>Intro</div>
+          We've built a few libraries to help you make Microsoft Teams-styled apps.
+          We hope that these libraries will enable seemless integration of your services into the
```

Comment:
"These give you controls, styles, and fonts that will easily make your apps have the look and feel common to other [Microsoft Teams apps]"